interpreter/text/lexer.mll

```diff
@@ -314,6 +314,14 @@ rule token = parse
   { SIMD_LOAD_LANE (fun a o i -> (v128_load32_lane (opt a 2)) o i) }
   | "v128.load64_lane"
   { SIMD_LOAD_LANE (fun a o i -> (v128_load64_lane (opt a 3)) o i) }
+  | "v128.store8_lane"
+  { SIMD_STORE_LANE (fun a o i -> (v128_store8_lane (opt a 0)) o i) }
```

Comment:
Nit: just noticed that the indentation of this and all previous semantic actions is off.